library/pom.xml

```diff
@@ -372,9 +377,39 @@
 							</execution>
 						</executions>
 					</plugin>
+
 				</plugins>
 			</build>
 		</profile>
+		<profile>
```

Comment:
By specific I mean like
${ckeditor-a11y-plugins} and ${ckeditor-a11y-extra-plugins}